perf/security/sds-tests/egress-sds/README.md

```diff
@@ -0,0 +1,46 @@
+# Test description
+
+This is a TLS egress gateway SDS test.
+The test sets up a number of TLS egress gateways for a group of nginx services.
+The test creates a group of sleep pods, where each sleep sends HTTP requests to a nginx
```

Comment:
nit: sleeping

Reply:
these are sleep pods not sleeping pods, sleep is more accurate to describe tem